WaysofWork.md

```diff
@@ -211,9 +217,19 @@ It allows to quickly view the total effort (as a sum of points) that each team h
 
 Likely to be the most day to day view, it shows issues by status.
 For simplicity only one view exists, where task level issues and stories cause confusion use the Story label to filter.
+By default it is filtered to show only tasks.
```

Comment:
What should be the role of the board in our day to day? Should it be used during the weekly? It could be a good way of centralising our activity

Reply:
I think it should be useful in the weekly and also daily as a regular tool for work.

If it isn't, then that is probably a problem.

Reply:
Issues need to be added to the project, which isnt happening. I have stressed the need in the issues section, also simplified fields so this can be done without much more work

Reply:
Yes, I'm definitely guilty of that.

I think some of the tension might be. Do I really have to add _every_ issue to the project board. Do I have to create an issue for everything I do that doesn't already have one?

It feels a bit like busywork where I'm not entirely sure what the benefit is. Especially if we report out "What we've been working on" to get an impression of what people are doing.

@jemrobinson @craddm @edwardchalstrey1 @harisood thoughts?

Reply:
Here is where I am split, honestly for me as RPM if you report out weekly I have most of what I need. I am still left without a way to assess actual workload (unless we complicate the weekly template) but probably not the end of the world.  But thought it would help us work better together and collaborate if we see what we are up to in a more dynamic, yet async, way. Think on what you want to use it for.

Also it is the best way to check that whatever we are working on contributes to the agreed priorities. If it doesn't match any, are we spending our time in what we said we wanted to spend it on?

So I would probably say yes to adding all issues if as a team we want to use the board, this can be progressive as you are about to update or create one just add the project.

Another solution for tasks that are not quite issues could be to only add them to stories, or to say in the weekly updated "contributed to story xx by doing yy" - Maybe if there is a check between work and goals that is enough where you think an issue would not be appropiate.